src/app/commands/cmd_sprite_size.cpp

```diff
@@ -180,17 +199,21 @@ class SpriteSizeJob : public SpriteJob {
 
 };
 
+#ifdef ENABLE_UI
+
 class SpriteSizeWindow : public app::gen::SpriteSize {
 public:
-  SpriteSizeWindow(Context* ctx, int new_width, int new_height) : m_ctx(ctx) {
+  SpriteSizeWindow(Context* ctx, SpriteSizeParams params) : m_ctx(ctx) {
```

Comment:
```suggestion
  SpriteSizeWindow(Context* ctx, const SpriteSizeParams& params) : m_ctx(ctx) {
```
Use `const SpriteSizeParams&` to avoid copying the `SpriteSizeParams` (we can use a reference to the original one from `SpriteSizeCommand`).

Reply:
Got it!